CPP_STYLE_GUIDE.md

```diff
@@ -0,0 +1,138 @@
+# C++ style guide
+
+Unfortunately, the C++ linter (based on
+[Google’s `cpplint`](https://github.com/google/styleguide)), which can be run
+explicitly via `make lint-cpp`, does not currently catch a lot of rules that are
+specific to the Node.js C++ code base. This document explains the most common of
+these rules:
+
+## Left-leaning (C++ style) asterisks for pointer declarations
+
+`char* buffer;` instead of `char *buffer;`
```

Comment:
No doubt this is correct, but it always bothers me with multiple variables:

    void* foo, * bar;

Reply:
I'd just declare one variable per line